src/parser/show_meta.c

```diff
@@ -444,9 +444,10 @@ metadata_of_index_capacity (SHOW_ONLY_ALL flag)
     {"Index_name", "varchar(256)"},
     {"Btid", "varchar(64)"},
     {"Num_distinct_key", "int"},
-    {"Total_value", "int"},
+    {"Total_value", "bigint"},
     {"Avg_num_value_per_key", "int"},
     {"Num_leaf_page", "int"},
+    {"Num_Ovfl_page", "bigint"},
```

Comment:
If the number of pages exceeds INT_MAX, it means that only the index is using 31 TiB.
Wouldn't it suffice to use INT yet?

Reply:
That's a good point.
(If the page size is 4K, it will be 8TiB.)
I agree with you. I'll fix it.